src/Auth0.AspNetCore.Authentication/Auth0WebAppAuthenticationBuilder.cs

```diff
@@ -10,16 +10,19 @@ public class Auth0WebAppAuthenticationBuilder
     {
         private readonly IServiceCollection _services;
         private readonly Auth0WebAppOptions _options;
+        private readonly string _authenticationScheme;
 
         /// <summary>
         /// Constructs an instance of <see cref="Auth0WebAppAuthenticationBuilder"/>
         /// </summary>
         /// <param name="services">The original <see href="https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.iservicecollection">IServiceCollection</see> instance</param>
+        /// <param name="authenticationScheme">The authentication scheme to use.</param>
         /// <param name="options">The <see cref="Auth0WebAppOptions"/> used when calling AddAuth0WebAppAuthentication.</param>
-        public Auth0WebAppAuthenticationBuilder(IServiceCollection services, Auth0WebAppOptions options)
+        public Auth0WebAppAuthenticationBuilder(IServiceCollection services, string authenticationScheme, Auth0WebAppOptions options)
```

Comment:
Wouldn't this change in parameter order be a breaking change? Or is this something used internally?

Reply:
I would say this is only used internally, but that wouldn't be true as it's public. We do not document using this, but would be breaking if anyone would.

Let me fix this and ensure this is backwards compatible with anyone using this.